src/rest_api/class-openagenda-controller.php

```diff
@@ -1768,6 +1768,21 @@ public function get_latlng_from_address( $address, $zipcode, $city ) {
 		$address_full = $address . ', ' . $zipcode . ' ' . $city;
 		$address_full = rawurlencode( $address_full );
 
+		// Set transient with the full address and check if it's the same as before or if it has been updated.
+		$transient_name_address = 'address_' . md5( $address_full );
+		$transient_get_address  = get_transient( $transient_name_address );
+		$transient_get_lat      = get_transient( 'latitude_' . md5( $address_full ) );
+		$transient_get_lon      = get_transient( 'longitude_' . md5( $address_full ) );
+		set_transient( $transient_name_address, $address_full );
```

Comment:
`set_transient` zonder een expiration wil zeggen dat hij nooit expired, weet je zeker dat je dat wilt? Want wat je dan (in het geval van geen external object caching) eigenlijk doet is een `add_option`

Bovendien wat is de toegevoegde waarde van deze transient? Je slaat datgene op wat je sowieso aangeleverd krijgt, er is geen processing (bijvoorbeeld een externe call) voor nodig om deze data te krijgen, dan lijkt caching mij niet heel nuttig?